static/app/components/group/tagFacets.tsx

```diff
@@ -15,6 +16,26 @@ import ButtonBar from '../buttonBar';
 import BreakdownBars from '../charts/breakdownBars';
 
 export const MOBILE_TAGS = ['os', 'device', 'release'];
+export function MOBILE_TAGS_FORMATTER(tagsData: Record<string, TagWithTopValues>) {
+  // For releases, format the release value to be more readable
```

Comment:
Nit: This comment should be closer to the `if (tagKey === 'release')` because if this gets extended, this comment will be inaccurate